### PR TITLE
feat(hierarchical): expose rootPath and showParentLevel

### PR DESCRIPTION
**Pull request by maxiloc**

### CODE REVIEW

**Comment by maxiloc** [comment]
finally :smile:

**Comment by vvo** [comment]
LGTM

File: src/widgets/hierarchical-menu/hierarchical-menu.js [hunk 1]
```diff
@@ -53,6 +57,8 @@ function hierarchicalMenu({
     container,
     attributes,
     separator = ' > ',
+    rootPath = null,
```
Comment (bobylito) [inline src/widgets/hierarchical-menu/hierarchical-menu.js]: Well it seems that setting null by default is not in fashion anymore. Could you change to no default value?

**Comment by vvo** [comment]
Still null so?

**Comment by maxiloc** [comment]
We can change it, if needed

**Pull request merged**